src/main/java/com/google/jspecify/nullness/NullSpecVisitor.java

```diff
@@ -579,11 +589,8 @@ private void checkNoNullnessAnnotations(
        * represent the types internally in AnnotatedTypeMirror instances. Contrast this to almost
        * all other logic in the checker, which operates on the internal types.
        */
-      if (areSameByName(annotation, "org.jspecify.annotations.Nullable")
-          || areSameByName(annotation, "org.jspecify.annotations.NullnessUnspecified")
-          || areSameByName(annotation, "org.jspecify.nullness.Nullable")
-          || areSameByName(annotation, "org.jspecify.nullness.NullnessUnspecified")) {
-        checker.reportError(treeToReportOn, messageKey);
+      if (NULLNESS_ANNOTATIONS.stream().anyMatch(na -> areSameByName(annotation, na))) {
```

Comment:
Could some of that TODO comment be cleaned up?

Reply:
I don't think so. It's all still relevant; I just moved the list of annotations to a constant (and added ~one~two).